_rules/1547.md

```diff
@@ -6,7 +6,7 @@ severity: 1
 ---
 Consider the following example:
 
-	if (member.HidesBaseClassMember && (member.NodeType != NodeType.InstanceInitializer))
+	if (member.HidesBaseClassMember && member.NodeType != NodeType.InstanceInitializer)
```

Comment:
🤔 I would keep the parentheses here.

Reply:
I'll revert these changes if you insist, but note it goes against the advice in https://github.com/dennisdoomen/CSharpGuidelines/blob/master/_rules/2400.md?plain=1#L55.

Reply:
You're right. There's no unclarity here.